draft-ietf-quic-recovery.md

```diff
@@ -1039,11 +1038,11 @@ Invoked by loss detection from DetectLostPackets when new packets
 are detected lost.
 
 ~~~
-   OnPacketsLost(lost_packets):
+   OnPacketsLost(lost_packet):
```

Comment:
Did you accidentally change lost_packets to lost_packet?

Reply:
Yeah, I shouldn't have made any changes there. Fixed it and squashed everything.